core-features/generate-client-and-cli-install-npm-lockfile/run.sh

```diff
@@ -4,11 +4,8 @@ set -eux
 
 VERSION=$(cat ../../.github/prisma-version.txt)
 
-# no install, instead rename file
-#pnpm install
-mv package.json package.json.backup
+cp -r ../_common/generate-client-and-cli-install/* .
 
```

Comment:
removing of lockfile not needed any more as before?

Reply:
Here I want to assert that the lockfile of the project is detected and used with the correct package manager.